lib/index.js

```diff
@@ -61,7 +64,7 @@ Intercom.prototype.identify = function(identify, fn){
       .set(headers(identify, self.settings))
       .type('json')
       .accept('json')
-      .send(mapper.identify(identify, { collectContext: collectContext }))
+      .send(mapper.identify(identify, {collectContext: collectContext, blacklisted: blacklisted, defaultMethod: defaultMethod, richLinkProperties: richLinkProperties }))
```

Comment:
nit pick, might be easier to read to define this as a `var` above like `options` or something and pass that in. kinda hard to read